models/issue_tracked_time.go

```diff
@@ -75,17 +104,43 @@ func (opts *FindTrackedTimesOptions) ToSession(e Engine) *xorm.Session {
 }
 
 // GetTrackedTimes returns all tracked times that fit to the given options.
-func GetTrackedTimes(options FindTrackedTimesOptions) (trackedTimes []*TrackedTime, err error) {
+func GetTrackedTimes(options FindTrackedTimesOptions) (trackedTimes TrackedTimeList, err error) {
 	err = options.ToSession(x).Find(&trackedTimes)
 	return
 }
 
+// GetTrackedSeconds return sum of seconds
+func GetTrackedSeconds(options FindTrackedTimesOptions) (trackedSeconds int64, err error) {
+	var trackedTimes TrackedTimeList
+	err = options.ToSession(x).Find(&trackedTimes)
+	if err != nil {
+		return 0, err
+	}
+	for _, t := range trackedTimes {
+		trackedSeconds += t.Time
+	}
+	return trackedSeconds, nil
+}
+
 // AddTime will add the given time (in seconds) to the issue
-func AddTime(user *User, issue *Issue, time int64) (*TrackedTime, error) {
+func AddTime(user *User, issue *Issue, amount int64) (*TrackedTime, error) {
```

Comment:
is this function still used? API module calls `AddTimeCreatedAt()`

Reply:
yes